app/controllers/api/account.php

```diff
@@ -1481,14 +1481,16 @@
             throw new Exception('Invalid credentials. User is blocked', 401); // User is in status blocked
         }
 
+        $expireTime = \time() + Auth::TOKEN_EXPIRATION_RECOVERY;
+
         $secret = Auth::tokenGenerator();
         $recovery = new Document([
             '$collection' => Database::SYSTEM_COLLECTION_TOKENS,
             '$permissions' => ['read' => ['user:'.$profile->getId()], 'write' => ['user:'.$profile->getId()]],
             'userId' => $profile->getId(),
             'type' => Auth::TOKEN_TYPE_RECOVERY,
             'secret' => Auth::hash($secret), // One way hash encryption to protect DB leak
-            'expire' => \time() + Auth::TOKEN_EXPIRATION_RECOVERY,
+            'expire' => $expireTime,
```

Comment:
Let's call this param only `$expire`, when choosing a naming convention we'll always prefer a single word when possible, and a leading word when a prefix is required. Although a bit pedantic, this helps us keep the code consistent, verbose and ready for future changes. Naming conventions are a big thing @ Appwrite 😅